src/domains/idpay-common/03_database.tf

```diff
@@ -487,6 +487,9 @@ locals {
         }, {
         keys   = ["trxChargeDate"]
         unique = false
+        }, {
```

Comment:
@frtrinca Could you move index update in another PR? It's a different terraform domain and cross-domain apply could rapidly get into a mess

Reply:
index moved in another PR